nixos/modules/programs/beidconnect.nix

```diff
@@ -0,0 +1,35 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+{
+  options.programs.beidconnect.enable = lib.mkEnableOption "beidconnect native messaging host";
```

Comment:
I am not sure we want a NixOS module for every individual native messaging host like this. It doesn’t scale very well to have these just to link some paths, and Firefox already has a native mechanism to include these messaging hosts (Chromium may too, I’m just not familiar with those options).

I would be more comfortable merging this with the previous version where it was just the package and considering NixOS integration later.